core/services/job/models.go

```diff
@@ -896,11 +896,12 @@ func (w *WorkflowSpec) Validate() error {
 }
 
 type StandardCapabilitiesSpec struct {
-	ID        int32
-	CreatedAt time.Time `toml:"-"`
-	UpdatedAt time.Time `toml:"-"`
-	Command   string    `toml:"command"`
-	Config    string    `toml:"config"`
+	ID            int32
+	CreatedAt     time.Time  `toml:"-"`
+	UpdatedAt     time.Time  `toml:"-"`
+	Command       string     `toml:"command"`
+	Config        string     `toml:"config"`
+	OracleFactory JSONConfig `toml:"oracleFactory"`
```

Comment:
@DeividasK OracleFactoryConfig? OracleFactory is a bit confusing. Also we presumably know the structure of this config? IIRC JSONConfig is just a map[string]any

Reply:
Can we put a comment here to document where we define what's expected in OracleConfig?